h/static/styles/logged-out-navbar.scss

```diff
@@ -0,0 +1,4 @@
+.logged-out-navbar {
+  margin-left: 30px;
+  margin-top: 15px;
```

Comment:
This is basically the amount of white margin space between the top and left edges of the document and any of our page content or elements. The same `30px` is repeated later on `.login-form` so that (if you make the browser window thin enough) the form aligns vertically with the h icon. This makes me think that this `30px` `margin/padding-left` ought to be on `<body>` or some containing element instead of duplicated.